aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -130,6 +130,10 @@ Tensor& deg2rad_out(Tensor& result, const Tensor& self) {
 Tensor deg2rad(const Tensor& self) { return unary_op_impl(self, at::deg2rad_out); }
 Tensor& deg2rad_(Tensor& self) { return unary_op_impl_(self, at::deg2rad_out); }
 
+Tensor degrees(const Tensor& self) { return at::native::rad2deg(self); }
```

Comment:
Comment documenting this is an alias of rad2deg. Might as well put these entries immediately under the rad2deg entries, too.